docs/configurations.md

```diff
@@ -0,0 +1,77 @@
+BALENA_HOST_FIREWALL_MODE | off | Toggle firewall modes between on, off, and auto. | v11.9.1
+BALENA_HOST_DISCOVERABILITY | true | Enable / Disable Avahi to run which will allow the device to respond to requests such as network scans. | v11.9.2
+BALENA_HOST_SPLASH_IMAGE | /boot/splash/balena-logo-default.png | Allows changing splash screen on boot to user defined file from userspace. | v12.3.0
+---
+
+In addition to these values, there may be some device-type specific configuration variables that can be set.
+
+## Config.txt
+
+> Supported boards: Any Raspberry Pi and BalenaFin.
+
+This device type utilizes a `config.txt` [file](https://www.raspberrypi.org/documentation/configuration/config-txt/README.md) which we offer some values to be configured.
+
+Name | Default | Description | Valid from
+--- | --- | --- | ---
+BALENA_HOST_CONFIG_disable_splash | 1 | Enable / Disable the Raspberry Pi rainbow splash screen. | v1.0.0
+BALENA_HOST_CONFIG_dtparam | "i2c_arm=on","spi=on","audio=on" | Define DT parameters. | v1.0.0
+BALENA_HOST_CONFIG_gpu_mem | 16 | Define device GPU memory in megabytes. | v1.0.0
+
+You can find more information on updating `config.txt` through configuration variables in our [Advanced Boot Configuration Guide][boot-config-guide].
+
+## ConfigFS
+
+> Supported boards: UP Board
+
+This device type utilizes a `configFS` [file](https://www.raspberrypi.org/documentation/configuration/config-txt/README.md) interface that allows definition of arbitrary functions and configurations.
+
+Name | Default | Description | Valid from
+--- | --- | --- | ---
+BALENA_HOST_CONFIGFS_ssdt | undefined | Allows configuring a user defined SSDTs from userspace via the configfs interface. See [SSDT Overlays](https://www.kernel.org/doc/html/latest/admin-guide/acpi/ssdt-overlays.html#ssdt-overlays). | v10.9.2
+
+## Extra-uEnv
+
+> Supported boards: Intel NUC, Nvidia Jetson Nano and TX2.
+
+This device type utilizes a file which we created to proxy configurations to [extlinux.conf](https://wiki.syslinux.org/wiki/index.php?title=Config).
+
+Name | Default | Description | Valid from
+--- | --- | --- | ---
+BALENA_HOST_EXTLINUX_isolcpus | undefined | Allows to isolate CPU cores from kernel scheduler.  See [kernel parameters](https://www.kernel.org/doc/html/latest/admin-guide/kernel-parameters.html?highlight=isolcpu#the-kernel-s-command-line-parameters). | v7.10.0
+BALENA_HOST_EXTLINUX_fdt | undefined | Allows configuring a user defined DTB from userspace. See [Device Tree Binary](https://elinux.org/Jetson/TX2_DTB). | v11.9.0
```

Comment:
Just to be clear for our users, we should probably mention that the device tree should be specified as a file name including extension (i.e device-tree.dtb), and not as a full path.